src/styles/components/pistol.scss

```diff
@@ -3,13 +3,74 @@
     height: 12rem;
     position: absolute;
     top: 1rem;
+    display: block;
+    border: 0;
+    background-color: transparent;
+    outline: 0;
+    cursor: crosshair;
 
+    img {
+        width: 12rem;
+        height: 12rem;
+    }
+    
     &--right {
         right: 9rem;
     }
+
+    &--right-animated {
```

Comment:
there could be done slight improvement to switch `&--is-right`, `&--is-left` etc